Java/Algorithms/kadanes_Java.java

```diff
@@ -0,0 +1,29 @@
+package DsAlgo;
+import java.util.*;
+public class kadanes_Java {
```

Comment:
@HarshAgarwal-DEV Use proper naming conventions for Classes, `public class Kadane` will work fine.